conversion.py

```diff
@@ -1193,30 +1209,32 @@ def decode_message(self, candidate, data, verify=True, allow_empty_signature=Fal
         if not self.can_decode_message(data):
             raise DropPacket("Cannot decode message")
 
+        # This method can return a deferred, so yield calls of it.
```

Comment:
After face to face discussion, we came to the conclusion that this comment can be a bit misleading. The `_decode_message_map` returns an object which methods can return deferreds.